v2.0/app/containers/MassEnergizeSuperAdmin/Testimonials/AllTestimonials.js

```diff
@@ -92,16 +92,17 @@ class AllTestimonials extends React.Component {
       key: 'rank',
       options: {
         filter: false,
-        customBodyRender: (d) => (
+        customBodyRender: (d) => d && (
           <TextField
+            key={d.id}
             required
             name="rank"
             variant="outlined"
             onChange={async event => {
               const { target } = event;
               if (!target) return;
               const { name, value } = target;
-              await apiCall('/testimonials.update', { testimonial_id: d && d.id, [name]: value });
+              await apiCall('/testimonials.rank', { testimonial_id: d && d.id, [name]: value });
```

Comment:
nit: if we have a whole route dedicated to updating rank then i think it makes sense to do 
```
{ testimonial_id: d && d.id, rank: value }
```
instead of 
```
{ testimonial_id: d && d.id, [name]: value }
```